front/style/pages/_register.scss

```diff
@@ -89,6 +89,22 @@
         &--flex {
             display: flex;
             align-items: center;
+
+            #form__mandate {
+                .select2-container {
+                    width: 100%;
+
+                    .select2-search-choice-close {
+                        border-bottom: unset;
```

Comment:
`unset` is not compatible with some browsers. (IE / IE Mobile & co.) 
See : https://caniuse.com/#feat=css-unset-value

You should use the default value, example for this property : 
`border-bottom: none;`

Reply:
:heavy_check_mark: